src/javascript/static/pages/job_details.js

```diff
@@ -30,7 +31,14 @@ const JobDetails = (() => {
         $sidebar.setVisibility(0);
         $('#title').find('h1').setVisibility(0);
         $('#image').find('img').setVisibility(0);
-        // show section
+
+        if ($sidebar_dept.length && window.location.hash === '') {
+            load(`${urlFor('open-positions/job-details')}?dept=${dept}${$sidebar_dept.find('a')[0].hash}`);
```

Comment:
can you use BinaryPjax.Load instead? I think we use it that way everywhere, it's easier to find that way